vscode-extensions.json

```diff
@@ -392,7 +392,7 @@
     },
     {
       "repository": "https://github.com/windup/rhamt-vscode-extension",
-      "revision": "617222def40678e2ef375f713b969c93ea313a69"
+      "revision": "v0.0.56"
```

Comment:
This doesn't appear to be a valid tag in the GitHub repo...a commit ID pointing to the `package.json` version bump is fine too.